CHANGES.md

```
@@ -1,3 +1,381 @@
+* Revert "Add configurable function and build size" in [9681](https://github.com/appwrite/appwrite/pull/9681)
+* core: introduce endpoint.docs in specs in [9685](https://github.com/appwrite/appwrite/pull/9685)
+* fix: remove content-type header from get request specs in [9666](https://github.com/appwrite/appwrite/pull/9666)
+* chore: update flutter sdk in [9691](https://github.com/appwrite/appwrite/pull/9691)
+
+# Version 1.6.1
+
+## What's Changed
+
+### Notable changes
+
+* Remove JPEG fallback for webp by @lohanidamodar in https://github.com/appwrite/appwrite/pull/8746
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)